### PR TITLE
[FIX] l10n_ar_bank: Agregamos los datos del banco BANCO BMA S.A.U.

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 76130
El banco Macro compro el banco ITAU

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/864).

File: l10n_ar_bank/data/res_bank.xml [hunk 1]
```diff
@@ -1046,4 +1046,16 @@
         <field name='phone'>011-4317-9000</field>
         <field name='email'>mariana.rodriguez@vwfs.com</field>
     </record>
+        <record model='res.bank' id='44089'>
```
Comment (zaoral) [inline l10n_ar_bank/data/res_bank.xml]: ident

**Review by zaoral — CHANGES_REQUESTED** [review]

**Review by zaoral — APPROVED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#865

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#865

**Pull request closed without merging**